source/slang/lower-to-ir.cpp

```diff
@@ -4012,18 +4012,19 @@ struct DeclLoweringVisitor : DeclVisitor<DeclLoweringVisitor, LoweredValInfo>
             // generating the code we actually care about, back in the original function.
 
             auto builder = getBuilder();
+
             auto initBlock = builder->createBlock();
             auto afterBlock = builder->createBlock();
 
             builder->emitIfElse(getSimpleVal(context, boolVal), afterBlock, initBlock, afterBlock);
 
-            builder->setInsertInto(initBlock);
+            builder->insertBlock(initBlock);
```

Comment:
This also reads better too. Not so sure about the 'emit' is create + insert naming, but I'd agree that's somewhat nit picking.

Reply:
Thoughts on an alternate naming convention? We currently use `emit*` for the common case because it matches the mental model of setting an insertion point and then emitting code in the order you want it to execute. The other operations are usually only needed when constructing non-sequential code.